akka-actor/src/main/scala/akka/dispatch/Dispatchers.scala

```diff
@@ -49,17 +51,28 @@ object Dispatchers {
    * configuration of the default dispatcher.
    */
   final val DefaultDispatcherId = "akka.actor.default-dispatcher"
+
+  final val InternalDispatcherId = "akka.actor.internal-dispatcher"
 }
 
 /**
  * Dispatchers are to be defined in configuration to allow for tuning
  * for different environments. Use the `lookup` method to create
  * a dispatcher as specified in configuration.
  *
+ * A dispatcher config can also be an alias, in that case it is a config string value pointing
+ * to the actual dispatcher config.
+ *
  * Look in `akka.actor.default-dispatcher` section of the reference.conf
  * for documentation of dispatcher options.
+ *
+ * Not for user instantiation or extension
  */
-class Dispatchers(val settings: ActorSystem.Settings, val prerequisites: DispatcherPrerequisites) {
+@DoNotInherit
+class Dispatchers(
+    val settings: ActorSystem.Settings,
+    val prerequisites: DispatcherPrerequisites,
+    logger: LoggingAdapter) {
```

Comment:
not binary compatible but we don't expect users to create own instances of this, maybe add `private[akka]` to the constructor at the the same time